libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/extensions/LocaleEx.kt

```diff
@@ -0,0 +1,24 @@
+@file:JvmName("LocaleEx")
+
+package com.mapbox.services.android.navigation.v5.utils.extensions
+
+import com.mapbox.api.directions.v5.DirectionsCriteria
+import java.util.Locale
+
+/**
+ * Returns the unit type for the specified locale. Try to avoid using this unnecessarily because
+ * all methods consuming unit type are able to handle the NONE_SPECIFIED type
+ *
+ * @return unit type for specified locale
+ */
+@DirectionsCriteria.VoiceUnitCriteria
+fun Locale.getUnitTypeForLocale(): String {
```

Comment:
Should we make it like 
```suggestion
fun Locale.getUnitTypeForLocale(): String = 
```